test/test.hs

```diff
@@ -2001,7 +2001,7 @@ tests = testGroup "hevm"
                     , "fullSuite/extcodelength.yul" -- extcodecopy bug?
                     , "loadResolver/keccak_short.yul" -- keccak bug
                     , "reasoningBasedSimplifier/signed_division.yul" -- ACTUAL bug, SDIV I think?
-                    ]
+                           ]
```

Comment:
indentation

Reply:
Hm, sorry, this was weird -- GHC complained about the indentation not matching the starting (`[`) bracket. So I moved it to the same column. But now it's happy to move it back. Weird, I'm still confused sometimes about indentation.

Reply:
Fixed!